app/src/main/java/com/google/jetpackcamera/MainActivity.kt

```diff
@@ -54,7 +54,7 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
         var uiState: MainActivityUiState by mutableStateOf(Loading)
 
-        // update viewmodel
+        // update view model
```

Comment:
I think we can just get rid of this comment.  It's inaccurate since we're updating the uiState here.